cms.php

```diff
@@ -30,40 +41,22 @@
     <link rel="stylesheet" type="text/css" href="cms.css">
     <title>Portfolio Input Page</title>
 </head>
-<h3>Welcome to the Portfolio Input Page</h3>
-
-<form method="post" action="cms.php">
-    <input type="text" value="<?php echo $email; ?>" name="email">email
-    <input type="submit" value="update">
-</form><br>
-<form method="post" action="cms.php">
-    <input type="tel" value="<?php echo $phone; ?>" name="tel">Telephone Number
-    <input type="submit" value="update">
-</form><br>
-<form method="post" action="cms.php">
-    <textarea rows="8" cols="150"  name="about" form="about">
-        <?php echo $about; ?></textarea>
-    <input type="submit" value="update">
-</form><br>
-
-    <h4>Project holder</h4>
-<?php
+<body>
+    <h3>Welcome to the About Me Input Page</h3>
+
+    <nav>
+        <a href="projects.php">Link to Projects Input Page</a><br><br>
+    </nav>
+
+    <form method="post" action="cms.php">
+        <input type="text" value="<?php echo $email; ?>" name="email_out">email
```

Comment:
Maybe `name` before `value`? It's makes more sense to me since keys become before values `[key => value]`